discord/flags.py

```diff
@@ -799,9 +798,7 @@ def guild_messages(self):
 
         .. note::
 
-            As of April 2022 requires opting in explicitly via the developer portal to receive the actual content of the messages.
-            Bots in over 100 guilds will need to apply to Discord for verification.
-            See https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Access-Deprecation-for-Verified-Bots for more information.
+            :attr:`message_content` is required to receive the actual content of guild messages.
```

Comment:
Should leave this, it's still valid 
```suggestion
            As of April 2022 requires opting in explicitly via the developer portal to receive the actual content of the guild messages.
            Bots in over 100 guilds will need to apply to Discord for verification.
            See https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Access-Deprecation-for-Verified-Bots for more information.
```